chrispinkney/cinesync
# Issue-93: Implement fetch utility functions

## Issue

Closes #93 

## Type of Change



## Description

This change is a refactor of the cinesync api fetches that are scattered across the application. Utility functions have been added into the `src/utils/` to allow fetches within components to call fetches with known parameters and known return types. 

This PR may need to be updated and resubmitted if PR #3 has been merged first.

## Testing the PR

Verify the following functionality works on preview:
- Sign up
- Sign in
- Open lists
- Create new list
- Open new list
- Add movie and save list
- Share list (may need to create second user)
- Delete movie
- Delete list
- Sign out

Optional if this functionality has been implemented:
- Upload user avatar
- Update user
- Delete user avatar
- Delete user

## Checklist
